isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -1077,30 +861,44 @@ namespace Isis {
    * @returns A list of the Cube Serial Numbers in the ControlNet.
    */
   QList< QString > ControlNet::GetCubeSerials() const {
-    return cubeGraphNodes->keys();
+    return m_vertexMap.keys();
   }
 
 
   /**
-   * @returns A list of all the cube graph nodes in the network
+   * Does a check to ensure that the given serial number is contained within
+   * the network.  If it is not, then an exception is thrown
```

Comment:
If the serial number is not contained in the network, this function returns false, it doesn't throw an exception.

Reply:
Basically this comment needs to be updated. Also @returns bool true if valid, otherwise false.